### PR TITLE
Remove bespoke codeowners

### PR DESCRIPTION
**Pull request by mvpatel2000**
# What does this PR do?

We still require reviews from @dakinggg and @eracah for loggers. This doesn't seem necessary and for some reason composer-team-eng approval w existing codeowners file is not enough.

Given it's not that big of a project code-wise, let's just strip bespoke codeowners.

### CODE REVIEW

**Review by dakinggg — APPROVED** [review]
I'm good with this, please wait for evan's approval as well.

**Review by eracah — APPROVED** [review]
LQGTM

**Pull request merged**
